tests/integration/test_client.py

```diff
@@ -0,0 +1,363 @@
+import asyncio
+import sys
+
+import pytest
+from bumble.device import Device
+from bumble.gatt import (
+    GATT_CHARACTERISTIC_USER_DESCRIPTION_DESCRIPTOR,
+    Characteristic,
+    Descriptor,
+    Service,
+)
+
+from bleak import BleakClient, BleakScanner
+from bleak.backends import BleakBackend, get_default_backend
+from bleak.backends.characteristic import BleakGATTCharacteristic
+from tests.integration.conftest import add_default_advertising_data
+
+if sys.version_info < (3, 11):
+    from async_timeout import timeout as async_timeout
+else:
+    from asyncio import timeout as async_timeout
+
+
+async def test_client_connect(bumble_peripheral: Device):
+    """Connecting to a BLE device is possible."""
+    add_default_advertising_data(bumble_peripheral)
+    await bumble_peripheral.power_on()
+    await bumble_peripheral.start_advertising()
```

Comment:
These 3 lines are getting repeated a bunch. Maybe time to make a utility function for it?